demo/settings_DE.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an example of a feature-complete JobFunnel configuration YAML.
+# Try this out by simply running: "funnel load -s demo/settings.yaml"
+
+# Path where your master CSV, block-lists, and cache data will be stored
+# NOTE: we create any missing directories in these filepaths
+master_csv_file: demo_job_search_results/demo_search.csv
+cache_folder: demo_job_search_results/cache
+block_list_file: demo_job_search_results/demo_block_list.json
+duplicates_list_file: demo_job_search_results/demo_duplicates_list.json
+log_file: demo_job_search_results/log.log
+
+# Job search configuration
+search:
+
+  # Locale settings, one of USA_ENGLISH, CANADA_ENGLISH, CANADA_FRENCH:
+  # This tells JobFunnel where the website we are scraping is located, and
+  # what language the contents are in.
+  locale: GERMANY_GERMAN
+
+  # Job providers which we will search, one of INDEED, MONSTER, GLASSDOOR:
+  # NOTE: we choose domain via locale (i.e. CANADA_ENGLISH -> www.indeed.ca)
+  # FIXME: we need to add back GLASSDOOR when that's working again.
+  providers:
+    - INDEED
+
+  # Region that we are searching for jobs within:
+  province_or_state: "108"  # NOTE: this is generally 2 characters long.
```

Comment:
can you elaborate on what this string is? I think the comment is wrong for this now, what does 108 refer to?

Reply:
You are right. I had mistaken this to be an internal indeed code. This variable actually does not make sense for the german site and it is also not used to build the search url. I updated the config and comment to highlight, that it is not used and loosened the config validation to allow for empty strings. The same also applies to the UK indeed scraper.